BlazarService/src/main/java/com/hubspot/blazar/config/BlazarConfiguration.java

```diff
@@ -37,6 +37,10 @@
   @JsonProperty("ui")
   private UiConfiguration uiConfiguration = new UiConfiguration();
 
+  @Valid
+  @JsonProperty("slack")
+  private SlackConfiguration slackConfiguration;
```

Comment:
I think this should be `Optional<SlackConfiguration>` and default to `Optional.absent();`